torch_tvm/compiler.cpp

```diff
@@ -69,14 +85,15 @@ tvm::relay::Expr TVMCompiler::convertToRelay(const IValue& val) {
 }
 
 tvm::relay::Function TVMCompiler::convertToRelay(
-    std::shared_ptr<Graph> subgraph) {
+    std::shared_ptr<Graph> subgraph, std::vector<Value*>* input_values) {
```

Comment:
just take a reference of the vector as argument?

Reply:
input_values is being written to, so I'm passing it as a pointer